setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import setup
 
 
-with open("vindaloo/vindaloo.py", "rt", encoding="utf8") as f:
+with open("vindaloo/vindaloo.py", "rt") as f:
     version = re.search(r'VERSION = \'(.*?)\'', f.read()).group(1)
```

Comment:
Nechtel bys mit soubor s konstantama, ktery by byl nezavisly a nemusel bys ho parsovat? A pak jen importnou verzi z toho souboru?

Reply:
Dobrej napad, zalozil bys na to issue?